src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide * secondSide / 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(
+                "Figure: right triangle, area: " + getArea() + " sq. units"
+                        + ", first side: " + firstSide + " units"
+                        + ", second side: " + firstSide + " units"

```

Comment:
You should use 'secondSide' instead of 'firstSide'. It seems like a copy-paste error.

Reply:
Indeed. In first step, I called them a, b and h instead of firstSide, secondSide and height. I am sure that at this time it was working correctly, because I checked it several times. But after I finished it maven asks about name repalcing. And in this operation for sure I  implemented this misaken code by copy and past wrong names.